### PR TITLE
Add steps to install from source for llama.cpp

### PR DESCRIPTION
**Pull request by Vaibhavs10**
ref: https://x.com/dnouri/status/1821197502760030686?s=46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,21 @@ You can quickly start a locally running chat-ui & LLM text-generation server tha
 
 **Step 1 (Start llama.cpp server):**
 
+Install llama.cpp w/ brew (for Mac):
```
Comment (julien-c) [inline README.md:40]: brew also exists for linux no?

**Review by julien-c — COMMENTED** [review]
oooh nice

File: README.md (hunk 1 above)
Comment (Vaibhavs10) [reply README.md:40]: It does, however, it only instals CPU only build, so would be quite slow. So recommended way would still be to `clone` + `make` imo.

**Comment by Vaibhavs10** [comment]
CI is 🟢

**Comment by dnouri** [comment]
I just found that the docs at llama.cpp have some really useful links for how to install on my platform.  Maybe you could add a link?

https://github.com/ggerganov/llama.cpp/blob/master/docs/build.md

**Comment by Vaibhavs10** [comment]
Yes! great suggestion @dnouri - I added the suggestions here in line 47:

```
or [build directly from the source](https://github.com/ggerganov/llama.cpp/blob/master/docs/build.md) for your target device:
```

**Review by nsarrazin — APPROVED** [review]
Looks great thanks!

**Pull request merged**